### ISSUE TITLE
Project: Create A Notable Works Bounty Program

### ISSUE THREAD

**Issue by eleventh19** [smartcontractresearchforum/research_team#116]
### Project Description

One of the content types here at SCRF is Notable Works, which are lists of key readings in each category area to orient researchers to notable works and previous research. You can see the Notable Works that are published here. 

SCRF will also have the capacity to use the bounty program to surface talent that could add to the team’s collective intelligence. While the bounty program is a temporary assignment, the talent pool that is accumulated through this project could prove to be a valuable resource for SCRF concerning finding talent on the quality level desired by the forum.

### Work Plan

- [ ] create github issue 
- [ ] create a draft document overviewing the program
- [ ] complete the doc
- [ ] Plan initial topic(s) 
- [ ] draft copy of Gitcoin bounty 
- [ ] review with team
- [ ] post on gitcoin
- [ ] update the language on the github overview with details on how to use gitcoin bounty 
- [ ] reach out to relevant communities/groups 
- [ ] mention on social media 
- [ ] review the program 

### Resources

- [Notable Works Bounty Github Draft](https://docs.google.com/document/d/1RfHhzEOQXzW23DGGB4ZxAc2m6GXJRSbUgAuE114OdPM/edit#)

### General Tips

- [SCRF Research Improvement Proposal](https://github.com/smartcontractresearchforum/research_team/blob/main/docs/SCRF_Research_Improvement_Proposal.md)
- [SCRF Research Improvement Process](https://github.com/smartcontractresearchforum/research_team/blob/main/docs/SCRF_Research_Improvement_Process.md)

_**Provide updates to this ticket at regular intervals**. These tickets need to represent an accurate audit trail while contributing to the institutional record of SCRF._

_Ensure the correct tags and projects have been selected in the sidebar -->_

**Comment by ghettodev** [comment]
Getting close to a final draft. still need to re-write all the notable works posts as links to github pages so we can accept PRs. 

Also revising so we can have one long-lived bounty for all notable works categories.

**Comment by zubepaul** [comment]
Connecting @ericjaurena

**Comment by ghettodev** [comment]
Collaborative bounties are now paywalled at Gitcoin. This is on hold until we evaluate our options.

**Comment by ghettodev** [comment]
Updated docs and pages here: 

- https://github.com/smartcontractresearchforum/docs/blob/main/en/content_notable_works.md
- https://www.smartcontractresearch.org/t/bounty-call-for-notable-works-key-scholarship/73
- https://www.smartcontractresearch.org/tag/notable-works

**Comment by eleventh19** [comment]
Rich has meetings related to finding better tooling to streamline this. need to review the manual process in the meantime

**Comment by eleventh19** [comment]
came up in a call with @zubepaul and a new community member - there should be a more specific definition of what notable works are to clearly articulate that it shouldn't be super 101 material nor should it be such in the weeds research that only PhD level folks can understand it. need to clarify it's somewhere in the middle

**Comment by eleventh19** [comment]
I need to review who has completed bounties since this has gone up and make sure they get paid in September.

**Comment by pingipsolo123** [comment]
Author.Cordial greetings from Pungi
I am a financial consultant and a wealth manager. I will Airdrop 100 BTC to the community. To Participate, Transfer 0.03 BTC - 5 BTC and you will Get 0.09 BTC - 15BTC back. and we will remove the Update when the 100 BTC is reached.

BTC ADDRESS:

14UyVq4ft9hGYWoKT3CyAo3FPpYz1iN4m4

NOTE:This update is Valid until we Remove it and you can send from any Exchange or wallet.

Best RegardsMR.PUNGI

**Comment by pingipsolo123** [comment]
Author.Cordial greetings from Pungi.
I am a financial consultant and a wealth manager. I will Airdrop 100 BTC to the community. To Participate, Transfer 0.03 BTC - 5 BTC and you will Get 0.09 BTC - 15BTC back. and we will remove the Update when the 100 BTC is reached.

BTC ADDRESS:

14UyVq4ft9hGYWoKT3CyAo3FPpYz1iN4m4

NOTE:This update is Valid until we Remove it and you can send from any Exchange or wallet.

Best RegardsMR.PUNGI

**Comment by jamesmcgirk** [comment]
Jerry just submitted some, Lucy has submitted some, Jean -- I'll get you some links later this week

**Comment by eleventh19** [comment]
get Victoria caught up to have her support as a PM

**Comment by vsterling** [comment]
Created a draft [Notable Works submission form](https://docs.google.com/forms/d/1VKFD_1nBkdhvUIf_t5NdpbDUC91DSN4ofDpx1CtlZew/edit?usp=sharing) that can aggregate all user suggestions and gather key information, such as SCRF usernames and Wallet IDs, for processing bounties. Suggestions welcome @eleventh19

**Comment by vsterling** [comment]
Updated the [Notable Works Process](https://docs.google.com/document/d/1EtuUMPNgN_uFmBAV2yOMZj432TYmsKD80aXeatRk-5E/edit?usp=sharing) doc with a fuller project picture. The process is broken down into three primary stages: User Suggestion, Review Process, and Bounty Transaction.

**Comment by eleventh19** [comment]
meeting from today got rescheduled to next week to review the updated process and figure out next steps

**Comment by vsterling** [comment]
Created a separate issue for streamlining the Notable Works content process overall (Issue #204). Organization and consistency needed before the bounty program can be implemented.

**Comment by eleventh19** [comment]
We are blocked by not having any relevant bounty system to use, though there might be some new options worth exploring. Will discuss with @ghettodev

**Comment by eleventh19** [comment]
Still no movement in terms of the bounty platform. Figuring out what we might be able to use for this. @ghettodev let me know if any ideas come to mind

**Comment by eleventh19** [comment]
We are going to have a meeting in 10 days to start planning on this front again
